charts/stan/values.yaml

```diff
@@ -0,0 +1,132 @@
+##################################
+#                                #
+#  NATS Streaming Configuration  #
+#                                #
+##################################
+stan:
+  image: nats-streaming:0.17.0
+  pullPolicy: IfNotPresent
+  replicas: 1
+
+  # Cluster name, generated into a name from the
+  # release name by default.
+  clusterID: 
+
+  logging:
+    debug: 
+    trace: 
+
+  # In case of NATS not being embedded then
+  # an URL for NATS is required.
+  nats:
+    url:
+    serviceRoleAuth:
+      enabled: false
+      natsClusterName:
+
+nameOverride: ""
+imagePullSecrets: {}
+
+# Toggle whether to use setup a Pod Security Context
+# ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+securityContext: null
+# securityContext:
+#   fsGroup: 1000
+#   runAsUser: 1000
+#   runAsNonRoot: true       
+
+# Affinity for pod assignment
+# ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+###########################
+#                         #
+#  Storage configuration  #
+#                         #
+###########################
+store:
+  # One value possible out of: "file" | "sql" | "memory"
+  type: file
+
+  # 
+  # Fault tolerance group
+  # 
+  ft:
+    group:
+
+  # Configures streaming/store_limits as is.
+  # 
+  # NOTE: Section in snake case since they will become
+  # the same objects.
+  # 
+  # Look at the NATS Streaming configuration for this:
+  # 
+  # https://docs.nats.io/nats-streaming-server/configuring/cfgfile#store-limits-configuration
+  # 
+  limits: {}
+
+  #
+  # Volume claim configuration. Required if the store type is `file` or if
+  # cluster is enabled.
+  #
+  volume:
+    enabled: true
+    mount: /data/stan
+    storageSize: 1Gi
+    accessModes: ReadWriteOnce
+    storageClass: 
+
+  # 
+  # File storage settings.
+  # 
+  file:
+    path: /data/stan/store
+
+  # In case of using file or sql storage, sets up a 3 node cluster.
+  cluster:
+    enabled: false
```

Comment:
I think we want this enabled so they will be clustered and share their state

Reply:
done